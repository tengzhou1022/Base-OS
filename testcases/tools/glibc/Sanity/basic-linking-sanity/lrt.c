#include <stdio.h>
#include <fcntl.h>

int main(){
  FILE *fp = fopen("lrt.out", "w");

  int opn = shm_open("/stuffz0r", O_RDWR|O_CREAT, 0777);
  if ( opn == -1 )
    fprintf(fp, "shm_open failed\n");
  else
    fprintf(fp, "shm_open successful\n");

  int unl = shm_unlink("/stuffz0r");
  if ( unl == -1 )
    fprintf(fp, "shm_unlink failed\n");
  else
    fprintf(fp, "shm_unlink successful\n");

  close(fp);
  return 0;
}
