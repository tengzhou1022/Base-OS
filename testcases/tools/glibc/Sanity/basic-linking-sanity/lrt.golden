shm_open successful
shm_unlink successful
